pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotAdministrator.java

```diff
@@ -192,7 +192,11 @@ public static void main(String[] args) {
     PluginManager.get().init();
     PinotAdministrator pinotAdministrator = new PinotAdministrator();
     pinotAdministrator.execute(args);
-    if (System.getProperties().getProperty("pinot.admin.system.exit", "false").equalsIgnoreCase("true")) {
+    // Ignore `pinot.admin.system.exit` property for Pinot quickstarts.
+    if ((args.length > 0) && ("quickstart".equalsIgnoreCase(args[0]))) {
```

Comment:
I know I'm late to the party, but why bury a somewhat obscure raw string comparison in this code (and elsewhere), versus having the quickstart code ensure `pinot.admin.system.exit` is set to `false`?